src/app/components/cells/cellCopyHelper.js

```diff
@@ -304,6 +317,7 @@ const pasteCellValue = function(
             newVals={newValue}
             saveAndClose={save}
             kind={dst.kind}
+            cell={src}
```

Comment:
```suggestion
            cell={dst}
```

Die zu-verändernde Zelle sollte doch "destination" sein, nicht "source", oder verwechsel ich grade was?